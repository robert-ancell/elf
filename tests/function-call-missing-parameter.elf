uint8 foo (uint8 value) {
   return value
}
foo ()
