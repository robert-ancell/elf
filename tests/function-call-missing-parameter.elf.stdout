Line 4:
foo ()
    ^
Function takes 1 parameters
